cmd/service/service.go

```diff
@@ -60,6 +76,9 @@ func makeRequest(method string, URL string, body io.Reader) (*http.Response, err
 	if resp.StatusCode == 404 {
 		return nil, fmt.Errorf("user not found")
 	}
+	if resp.StatusCode == 500 {
```

Comment:
In the latest changes, If the status code is not 200. we just return status whatever we get.